de/api/configuration-ignore.md

```diff
@@ -3,6 +3,36 @@ title: "API: The ignore Property"
 description: Define the ignore files for your Nuxt.js application
 ---
 
+# .nuxtignore
```

Comment:
Why adding it to de as well? :thinking:

Reply:
The original pr is opened by adding de incorrectly, I pushed en just now.

Since this page in de is English, do I need to remove it ?

Reply:
ah!

no, you can keep it :relaxed: